### PR TITLE
Update documentation according to new Ank CLI behaviour

### PR DESCRIPTION
**Pull request by christoph-hamm**
Issues: #13

As the behavior of the Ank CLI has changed (it waits for the workload to be created/deleted) and also it outputs changed, the documentation needs to be updated.  

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [ ] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

File: doc/docs/usage/tutorial-vehicle-signals.md [hunk 1]
```diff
@@ -184,13 +173,22 @@ ank apply --agent infotainment speed-consumer.yaml
     ank apply --agent infotainment speed-consumer.yaml
     ```
 
-And we check that the new workload is running:
+The command wait until speed consumer is running.
```
Comment (krucod3) [inline doc/docs/usage/tutorial-vehicle-signals.md:176]:
```suggestion
The command waits until speed consumer is running.
```

File: doc/docs/usage/tutorial-vehicle-signals.md [hunk 2]
```diff
@@ -184,13 +173,22 @@ ank apply --agent infotainment speed-consumer.yaml
...
+It should print:
+
```
Comment (krucod3) [inline doc/docs/usage/tutorial-vehicle-signals.md:178]:
```suggestion
```

File: doc/docs/usage/tutorial-vehicle-signals.md [hunk 3]
```diff
@@ -184,13 +173,22 @@ ank apply --agent infotainment speed-consumer.yaml
...
+```
+
+We can check all running workloads with
 
 ```shell
 ank get workloads
 ```
 
-With the result:
+With test result:
```
Comment (krucod3) [inline doc/docs/usage/tutorial-vehicle-signals.md:191]:
```suggestion
The output should be:
```

**Review by krucod3 — COMMENTED** [review]
Some minor suggestions.

File: doc/docs/usage/tutorial-vehicle-signals.md [hunk 4]
```diff
@@ -57,17 +57,12 @@ ank apply databroker.yaml
...
 The Ankaios agent `agent_A` will now instruct podman to start the workload.
-We can watch the result with:
-
-```shell
-ank get workloads
-```
-
-That command shall return:
+The command waits until databroker is running.
+It should print:
```
Comment (windsource) [inline doc/docs/usage/tutorial-vehicle-signals.md:61]:
```suggestion
The command waits until the databroker is running.
It should finally print:
```

File: doc/docs/usage/tutorial-vehicle-signals.md [hunk 5]
```diff
@@ -94,18 +89,12 @@ Start the workload with:
...
 
-To check that the workload has been started call:
-
-```shell
-ank get workloads
-```
-
-With the result:
+The command waits until speed provider is running.
+It should print:
```
Comment (windsource) [inline doc/docs/usage/tutorial-vehicle-signals.md:93]:
```suggestion
The command waits until the speed-provider is running.
It should finally print:
```

**Review by windsource — APPROVED** [review]
LGTM 👍

**Pull request merged**